packages/@textlint/markdown-to-ast/test/fixtures/deletion.text/output.json

```diff
@@ -6,80 +6,22 @@
             "children": [
                 {
                     "type": "Str",
-                    "value": "hello ",
+                    "value": "hello ~~hi~~ world",
```

Comment:
hello ~~hi~~ world

- [ ] It is a bug?

GitHub render it as Delete.

Reply:
https://github.com/remarkjs/remark-gfm#optionssingletilde

It was moved to option.